src/main/java/com/mongodb/jdbc/MongoResultSet.java

```diff
@@ -93,36 +98,22 @@ MongoResultDoc getCurrent() {
         return current;
     }
 
+	private BsonValue getBsonValue(int columnIndex) throws SQLException {
```

Comment:
had to change all the lines in each `getX` function that were getting the column index from the label because the column position cache was moved to the ResultSetMetadta. While I was changing that, I factored this logic out into a single helper function, so that it's not duplicated in each getter